### PR TITLE
fix JobCreateCommand: don't overwrite capabilities from json file

### PR DESCRIPTION
**Pull request by mattnworb**
Added or dropped capabilities from a job json file were overwritten if the CLI switch had an empty value (i.e. was not specified).

### CODE REVIEW

**Comment by mattnworb** [comment]
@davidxia @gimaker @negz

File: helios-integration-tests/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -9,7 +9,6 @@
   </appender>
 
   <logger name="com.spotify.helios.client" level="info"/>
-  <logger name="com.spotify.hermes" level="info"/>
```
Comment (gimaker) [inline helios-integration-tests/src/test/resources/logback-test.xml]: spurious change, or intentional?

Comment (mattnworb) [inline helios-integration-tests/src/test/resources/logback-test.xml]: intentional; why was this line ever here? No code in helios integration tests has this package name.

My guess is it was copy & pasted from somewhere else.

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java [hunk 2]
```diff
@@ -546,8 +546,15 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
       builder.setToken(token);
     }
 
-    builder.setAddCapabilities(options.<String>getList(addCapabilityArg.getDest()));
-    builder.setDropCapabilities(options.<String>getList(dropCapabilityArg.getDest()));
+    final List<String> addCaps = options.<String>getList(addCapabilityArg.getDest());
```
Comment (gimaker) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: For port mappings and registrations, it seems like we merge the values if they come from more than one source. Perhaps do the same here for consistency?

File: helios-integration-tests/src/test/resources/logback-test.xml (hunk 1 above)
Comment (gimaker) [inline helios-integration-tests/src/test/resources/logback-test.xml]: If you're cleaning up, maybe remove `<logger name="com.spotify.helios.client" level="info"/>` as well then?

**Comment by gimaker** [comment]
:+1:, comments aside

File: helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java (hunk 2 above)
Comment (mattnworb) [inline helios-tools/src/main/java/com/spotify/helios/cli/command/JobCreateCommand.java]: I don't have a strong preference, just trying to fix a broken part of the current implementation

**Comment by mattnworb** [comment]
cc @mzeo

**Pull request merged**